plugins/advisors/black-duck/src/main/kotlin/BlackDuck.kt

```diff
@@ -220,6 +224,18 @@ internal fun VulnerabilityView.toOrtVulnerability(): Vulnerability {
     )
 }
 
+private fun VulnerabilityCvss3View.getScoringSystemAndVector(): Pair<String, String> {
+    val scoringSystem = vector.substringBefore('/', "").ifEmpty { Cvss3Rating.PREFIXES.first() }
+
```

Comment:
Consider dropping this empty line now to visually align with the function below.